src/en/clouds.md

```diff
@@ -161,29 +164,36 @@ expand the relevant section). You can also generate a YAML file
 
 ^# Manual
 
-   To add a 'manual' cloud, Juju only needs to know the name you wish to call it, and 
-   the network address used to connect to it. A sample session looks like this:
-       
+   To add a Manual cloud, Juju needs to know the name you wish to call it, the
+   IP address (or hostname) used to connect to it, and what remote user account
+   to connect to (over SSH). This last is done by prepending 'user@' to the
+   address/hostname.
+   
+   In terms of SSH, the user running the Juju client is expected to already be
+   able to connect to the remote host (either by password or public key).
+   
+   A sample session looks like this:
+
        juju add-cloud
-  
+
        Cloud Types
         maas
         manual
         openstack
         oracle
         vsphere
-
-       Select cloud type: maas
-
-       Enter a name for your maas cloud: mainmaas
-
-       Enter the API endpoint url: http://maas.example.org:5240/MAAS/api/2.0
-
-      Cloud "mainmaas" successfully added
-      You may bootstrap with 'juju bootstrap mainmaas'
-
-   Once completed, you should also remember to add a credential for this cloud before 
-   bootstrapping. See the [documentation on credentials][credentials] for more help.
+      
+      Select cloud type: manual
+      
+      Enter a name for your manual cloud: mycloud
+      
+      Enter the controller's hostname or IP address: noah@10.143.211.93
+      
+      Cloud "mycloud" successfully added
+      You may bootstrap with 'juju bootstrap mycloud'
+
+   You must now add a credential for this cloud prior to creating a controller
```

Comment:
You mentioned adding language like this for other clouds, but I don't think it needs to be said here for a manual cloud.

Reply:
Gah, I knew that and forgot! :+1: